documentation/asciidoc/accessories/iqaudio/configuration.adoc

```diff
@@ -0,0 +1,134 @@
+== Configuration
+
+All our latest full size audio boards include a pre-programmed EEPROM. This allows Raspberry Pi
+OS to auto detect and auto configure itself allowing the IQaudio boards to be plug and play.
+
+If you need to configure Raspberry Pi OS yourself, you can edit the boot configuration file (found at
+`/boot/config.txt`) to manually add the appropriate device-tree overlay:
+
+|===
+| Board | *`/boot/config.txt`*
+
+| DAC Pro | `dtoverlay=iqaudio-dacplus`
+| DAC{plus} | `dtoverlay=iqaudio-dacplus`
+| DigiAMP{plus} | `dtoverlay=iqaudio-dacplusdtoverlay=iqaudio-dacplus,unmute_amp` or `dtoverlay=iqaudio-dacplus,auto_mute_amp`
+| Codec Zero | `dtoverlay=iqaudio-codec`
+|===
+
+=== Disabling on-board audio
+
+To make your IQaudio sound card the primary audio device in Raspberry Pi OS you will need to
+disable the Raspberry Pi’s on-board audio card. This is done simply by commenting out the
+`dtparam=audio=on`device tree parameter on your sd card’s `/boot/config.txt` file by placing a `# at the
+start of the line, saving the file and rebooting your Pi.
+
+``/boot/config.txt`` should have the following entry to disable the Pi’s on-board sound card.
+
+----
+#dtparam=audio=on
+----
+
+=== Headless software
+
+Many of the software packages used with the Raspberry Pi simply turn the Pi into a “Headless” endpoint. That is to say that it doesn't need, or expect, a screen/display to be physically attached to the Raspberry Pi’s HDMI socket
+
+So if there’s no screen how do you control it?
+
+Well in some cases you don’t need to do anything with the Pi at all. Take the https://roonlabs.com/[RoonLabs] images, that
+turns the Raspberry Pi into a Roon Certified end-point. The Pi is then controlled form the RoonLabs
+PC/Mac application which connects to and plays music through the IQaudio sound card on your
+Raspberry Pi. No screen needs to be attached to the Pi itself.
```

Comment:
Why don't you get rid of the line change? and

to the Pi itself. -> to the Raspberry Pi itself.